new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.js

```diff
@@ -32,6 +34,12 @@ const FiatBalance = ({
   const editing = isEditing(NAME)
 
   const schema = Yup.object().shape({
+    /* cashInAlertThreshold: Yup.number()
```

Comment:
Can we delete all the commented out code in this PR?

Reply:
Should be all removed by now. Future PRs introducing the cashbox to the alerts and balance override, should be able to reuse the commented code